main.py

```diff
@@ -1,12 +1,277 @@
+        "3": "Select a video",
+        "4": "Edit selected video",
+        "5": "Delete selected video", 
+        "6": "List all customers",
+        "7": "Add a customer",
+        "8": "Select a customer",
+        "9": "Update selected customer",
+        "10": "Delete selected customer",
+        "11": "Check out a video to a customer",
+        "12": "Check in a video from a customer",
+        "13": "List all options",
+        "14": "Quit"
+        }
+
+    print_frogges()
+    print("WELCOME TO RETRO VIDEO STORE\n")
+    print("These are the actions you can perform:")
+    print_frogges()
+    
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_frogges()
+
+    return options
+
+
+def make_choice(options, customer, video):
+    """
+    input: the list of options from list_options()
+    output: 
+    """
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("\nWhat would you like to do? Select 13 to see all options again, OR...")
+        choice = input("\nMake your selection using the option number: ")
+
+    if choice in ['4','5'] and video.selected_video == None:
+        print("\nYou must select a video before updating it or deleting it.")
+        print("\nLet's select a video!")
+        choice = "3"
+
+    if choice in ['9', '10'] and customer.selected_customer == None:
+        print("\nYou must select a customer record before updating it or deleting it.")
+        print("\nLet's select a customer!")
+        choice = "8"
+    
+    return choice
+    
+
+def run_cli(play=True):
+    """
+    input
+    output
+    """
+
+    customers_list = Customer()
+    videos_list = Video()
+    rental_action = Rental()
+
+    options = list_options()
+
+    while play==True:
+
+        choice = make_choice(options, customers_list, videos_list)
+
+        customers_list.print_selected_customer()
+        videos_list.print_selected_video()
+
+        # "List all videos"
+        if choice=='1':
+            print_frogges()
+            for video in videos_list.get_all_videos():
+                print("ID: ", video["id"])
+                print("Title: ", video["title"])
+                print("Release date: ", video["release_date"])
+                print("Total inventory: ", video["total_inventory"])
+                print("Available inventory: ", video["available_inventory"])
+                print("\n٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ ٭ \n")
+
+        # "Add a video"
+        elif choice=='2':
+            title = input("\nEnter video title:\n")
+            release_date = input("\nEnter video release date: ")
+            total_inventory = input("\nEnter total starting inventory: ")
+            response = videos_list.add_video(title=title, release_date=release_date, total_inventory=total_inventory)
+
+            print_frogges()
+            print("\nSuccessfully added new video with ID:", response["id"])
```

Comment:
When the API responds with an error, this line will trigger an exception to be thrown because there won't be an "id" in the response. See my comment on the add_video method for more details